holesky/README.md

```diff
@@ -47,7 +47,19 @@ In order to limit traffic from the EigenLabs hosted Disperser, please restrict y
 - `34.232.117.230/32`
 - `18.214.113.214/32`
 
-#### Step 5: Opt-in into EigenDA
+
+#### Step 5: Quorum Configuration
```

Comment:
I'm not sure if this should be a step: the operator is not expected to do anything at this step.

Reply:
@jianoaix  I'm open to your preference.

IMO there are benefits to us having the user make an intentional (explicit) choice of which quorum to configure and opt-into via .env.  It would ensure their .env configuration matches the WETH or ETH delegation TVL they planned for (intended) via the Operator web app UI and reduce potential for confusion.

Your preference?

Reply:
Ok sounds fine, so the action for operator at this step is basically setting quorums you want to opt-in.

One thing important to note for operators is that they can only set quorums that they are currently **not** registered in the NODE_QUORUM_ID_LIST (e.g. if they are already in 0, and want to opt-in one more quorum 1, they have to set this flag to "1", not "0,1"). I think this guidance was around but somehow got deleted.

Similarly, when opt-out, quorums specified in NODE_QUORUM_ID_LIST has to be quorums they **already** registered in.

Reply:
Good call out. I've updated the text just now to include the following:

> You only set quorums that you are currently not registered to in the NODE_QUORUM_ID_LIST. For example if you are already registered to quorum 0 and want to opt-in one more quorum 1, then you must set NODE_QUORUM_ID_LIST to `1` (not `0,1`).

My sense is we will need to improve this section with more detail in the coming days